tajo-core/src/main/java/org/apache/tajo/querymaster/Repartitioner.java

```diff
@@ -725,13 +725,8 @@ public static void scheduleRangeShuffledFetches(TaskSchedulerContext schedulerCo
       }
     }
 
-    boolean ascendingFirstKey = sortSpecs[0].isAscending();
     SortedMap<TupleRange, Collection<FetchImpl>> map;
-    if (ascendingFirstKey) {
-      map = new TreeMap<TupleRange, Collection<FetchImpl>>();
-    } else {
-      map = new TreeMap<TupleRange, Collection<FetchImpl>>(new TupleRange.DescendingTupleRangeComparator());
```

Comment:
This patch looks good to me.  
@hyunsik and @jihoonson Do we have any reason  to keep the descending ranges ?